tests/utils/managed_deployment.py

```
@@ -0,0 +1,768 @@
+    def wait(self, timeout: int = 60):
+        """Wait for this process to exit in the given pod"""
+        # Simple implementation; adjust as needed
+        for _ in range(timeout):
+            try:
+                result = self._pod.exec(
+                    ["kill", "-0", str(self.pid)]
+                )  # Check if process exists
+                if result.returncode != 0:
+                    return True  # Process exited
+                time.sleep(1)
+            except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)